src/OpenTelemetry.Exporter.Prometheus/PrometheusExporterExtensions.cs

```diff
@@ -217,5 +109,69 @@ public static string GetMetricsCollection(this PrometheusExporter exporter)
 
             return Encoding.UTF8.GetString(stream.ToArray(), 0, (int)stream.Length);
         }
+
+        private static void WriteSum(StreamWriter writer, PrometheusMetricBuilder builder, IEnumerable<KeyValuePair<string, string>> labels, double doubleValue)
+        {
+            builder = builder.WithType(PrometheusCounterType);
+
+            foreach (var label in labels)
+            {
+                var metricValueBuilder = builder.AddValue();
+                metricValueBuilder = metricValueBuilder.WithValue(doubleValue);
+                metricValueBuilder.WithLabel(label.Key, label.Value);
+            }
```

Comment:
Isn't this going to generate one time-series per each label pair? I mean it seems that what is wanted is to have something like the suggestion below - am I missing something here?
```suggestion
            var metricValueBuilder = builder.AddValue();
            metricValueBuilder = metricValueBuilder.WithValue(doubleValue);
            foreach (var label in labels)
            {
                metricValueBuilder.WithLabel(label.Key, label.Value);
            }
```

Reply:
I did a quick test using a [modified sample console](https://github.com/open-telemetry/opentelemetry-dotnet/compare/master...pjanotti:test-prom-exporter?expand=1) app:
```terminal
# HELP MyCounterMyMeterMyCounter
# TYPE MyCounter counter
MyCounter{dim1="value1"} 1500 1588112149908
MyCounter{dim2="value2"} 1500 1588112149908
# HELP MyMeasureMyMeterMyMeasure
# TYPE MyMeasure summary
MyMeasure_sum{dim1="value1"} 20325 1588112149909
MyMeasure_count{dim1="value1"} 60 1588112149909
MyMeasure{dim1="value1",quantile="0"} 5 1588112149909
MyMeasure{dim1="value1",quantile="1"} 750 1588112149909
MyMeasure_sum{dim2="value2"} 20325 1588112149909
MyMeasure_count{dim2="value2"} 60 1588112149909
MyMeasure{dim2="value2",quantile="0"} 5 1588112149909
MyMeasure{dim2="value2",quantile="1"} 750 1588112149909
# HELP MyObservationMyMeterMyObservation
# TYPE MyObservation counter
MyObservation{dim1="value1"} 28704768 1588112149909
```

Reply:
Yes! thanks for catching that. I modified unit test to have multiple labels, and validated the fix.